no-computer.md

```diff
@@ -11,41 +11,93 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+The idea is to have a story made out of modular pieces that could all
+fit together to form something of a syntactically correct, but potentially
+nonsensical narrative.  The purpose of the idea would be for entertainment.
```

Comment:
Fun! Reminds me a bit of the game where each person takes turn saying a word and you try to make something that makes some senes but you can also kinda mess with the other person by setting them up to have a difficult time saying a word that will make sense after yours.